nautobot/core/tests/test_utils.py

```diff
@@ -373,17 +373,20 @@ def test_slugify_dashes_to_underscores(self):
 
 class LookupRelatedFunctionTest(TestCase):
     def test_is_single_choice_field(self):
-        # Assert function returns True for any field starting with create or has_
-        # Cause these fields are either boolean fields or date time fields which one accepts single values
+        """
+        Assert that is_single_choice_field() correctly distinguishes between single-value and multi-value filter fields.
+        """
         filterset_class = dcim_filters.LocationFilterSet
 
-        single_choice_fields = ("created", "created__gte", "has_vlans", "has_clusters", "q")
+        single_choice_fields = ("has_vlans", "has_clusters")
         for field in single_choice_fields:
-            self.assertTrue(requests.is_single_choice_field(filterset_class, field))
+            with self.subTest(f"Single choice field: {field}"):
+                self.assertTrue(requests.is_single_choice_field(filterset_class, field))
 
-        multi_choice_fields = ("status", "tenant", "tags")
+        multi_choice_fields = ("created", "status", "tenant", "tags")
```

Comment:
Why are we not testing `created__gte` and `q` fields anymore?

Reply:
I removed `q` because of #1623 (it probably *shouldn't* be a single-value filter), and `created__gte` is no longer a special-case filter.